centml/compiler/config.py

```diff
@@ -31,5 +31,7 @@ class Config(BaseSettings):
     # If the server response is smaller than this, don't gzip it
     MINIMUM_GZIP_SIZE: int = 1000
 
+    PREDICTING: bool = True
```

Comment:
instead of a boolean flag, can we use an Enum to choose different backends?

In the future, we might have more backends like compile and predict.